### DISCUSSION TITLE
program languages源代码语言

### DISCUSSION THREAD

**Discussion by smerra1127** [Meloong-Git/PCL#2278] [讨论]
VB语言看不懂 能够翻成其他语言吗 比如java python 研究学习下
Can VB language be translated into other languages if I don't understand it

**Comment by so1ve** [comment]
java的话去看hmcl源代码，vb看个语法大概就能看懂了，至于py，，，紫菜罢（）

实在不行你找一段看不懂的代码丢给gpt去解释

**Reply by smerra1127** [reply]
gpt问题太多
看很多up主测试gpt写代码效果不理想

**Reply by so1ve** [reply]
GPT简单理解个代码问题还是不大的，只要你上下文给足

**Reply by smerra1127** [reply]
这就试试

**Comment by Chiloven945** [comment]
gtp可以转代码来着，只是看的话估计没问题
~还有启动器的源码都是屎山~

**Reply by so1ve** [reply]
这可不一定，xmcl的源代码很优雅的（）

**Comment by smerra1127** [comment]
有好用的源码解码器吗
